docs/book/installation/sylius_plus_installation.rst

```diff
@@ -94,6 +94,7 @@ Installing Sylius Plus as a plugin to a Sylius application
 * Order
 * ProductVariant
 * Shipment
+* ReturnRequest
```

Comment:
IMO it should be at a new point, this point: `Add traits that enhance Sylius models` is related to models from Sylius, but Return Request is a model from Plus